contracts/cw-proposal-multiple/src/contract.rs

```diff
@@ -414,6 +419,8 @@ pub fn execute_close(
     };
 
     prop.status = Status::Closed;
+    // Update proposal's last updated timestamp.
+    prop.last_updated = env.block.time;
```

Comment:
I see that we did the same thing in the single-choice PR, but Im just wondering why we didn't do this `prop.last_updated` update here as well?: https://github.com/DA0-DA0/dao-contracts/blob/main/contracts/cw-proposal-multiple/src/contract.rs#L750